pkg/command/plugin.go

```diff
@@ -343,8 +343,9 @@ func newUpgradePluginCmd() *cobra.Command {
 
 func newDeletePluginCmd() *cobra.Command {
 	var deleteCmd = &cobra.Command{
-		Use:               "delete " + pluginNameCaps,
-		Short:             "Delete a plugin",
+		Use:               "uninstall " + pluginNameCaps,
+		Aliases:           []string{"delete"},
+		Short:             "Uninstall a plugin",
```

Comment:
What do you think of replacing `delete` in the two success messages further down?

Reply:
Good catch. Updated.